src/tracks/Pattern.cpp

```diff
@@ -789,6 +790,10 @@ void PatternView::mouseDoubleClickEvent( QMouseEvent * _me )
 	  			m_pat->m_steps != MidiTime::stepsPerTact() ) &&
 		_me->y() > height() - s_stepBtnOff->height() ) )
 	{
+		if( m_lastClickChange.msecsTo( QTime::currentTime() ) < QApplication::doubleClickInterval() )
```

Comment:
I am talking for me: a comment describing why this logic is required would be really helpful for people who find out this code later.

Something like: `// Avoid changing BB notes on double-click`